WordPress/src/main/java/org/wordpress/android/ui/accounts/BlogUtils.java

```diff
@@ -188,4 +188,20 @@ public static void convertToLowercase(Editable s) {
             s.replace(0, s.length(), lowerCase);
         }
     }
+
+    public static Set<String> planTags() {
+        List<Map<String, Object>> blogs = WordPress.wpDB.getBlogsBy("dotcomFlag=1",new String[]{"plan_product_id"});
+        Set<String> tags = new HashSet<String>();
```

Comment:
The type parameter for the `HashSet` is redundant in this case. I'd suggest removing it, leaving just the empty `<>`. See [type inference on Java 7+](http://docs.oracle.com/javase/7/docs/technotes/guides/language/type-inference-generic-instance-creation.html) for more information. Thanks!